docs/apache-airflow-providers-salesforce/operators/salesforce_bulk.rst

```diff
@@ -0,0 +1,55 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+.. _howto/operator:SalesforceBulkOperator:
+
+
+SalesforceBulkOperator
+======================
+
+Use the :class:`~airflow.providers.salesforce.operators.salesforce_bulk.SalesforceBulkOperator` to execute Bulk API.
+
+Using the Operator
+^^^^^^^^^^^^^^^^^^
+
+You can use this operator to access Bulk Insert API:
+
+.. exampleinclude:: /../../tests/system/providers/salesforce/example_salesforce_bulk.py
+    :language: python
```

Comment:
```suggestion
    :language: python
    :dedent: 4
```
Since the `START` marker is indented we should use the `:dedent:` param for rendering the code snippet appropriately.